### ISSUE TITLE
Import Failed while installing through manager

### ISSUE THREAD

**Issue by manassm** [MrForExample/ComfyUI-3D-Pack#237]
I apologize in advance for my lack of computer knowledge (especially linux, which I am new to).

I was trying to install ComfyUI-3D-Pack through comfyui manager, but I have been getting errors and was not able to.
I am not sure if any of the setup I have is causing the problem, especially since I installed my version of ComfyUI through Pinokio on Linux Mint 21.3 (which I believe is Ubuntu 22.04.3).
I am using Nvidia RTX4080 with Intel CPU.

I would highly appretiate it if someone could guide me through resolving the situation.
The following is the terminal log on bootup:
```
<<PINOKIO SHELL>>eval "$(conda shell.bash hook)" && conda deactivate && conda deactivate && conda deactivate && conda activate base && source /home/sephia/pinokio/api/comfyui.git/app/env/bin/activate /home/sephia/pinokio/api/comfyui.git/app/env && python main.py --disable-xformers
[START] Security scan
[DONE] Security scan
## ComfyUI-Manager: installing dependencies done.
** ComfyUI startup time: 2024-07-23 22:17:34.343217
** Platform: Linux
** Python version: 3.10.14 | packaged by conda-forge | (main, Mar 20 2024, 12:45:18) [GCC 12.3.0]
** Python executable: /home/sephia/pinokio/api/comfyui.git/app/env/bin/python
** ComfyUI Path: /home/sephia/pinokio/api/comfyui.git/app
** Log path: /home/sephia/pinokio/api/comfyui.git/app/comfyui.log

Prestartup times for custom nodes:
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/rgthree-comfy
   0.4 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-Manager

Total VRAM 16071 MB, total RAM 31863 MB
pytorch version: 2.3.0+cu121
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4080 : cudaMallocAsync
Using pytorch cross attention
[Prompt Server] web root: /home/sephia/pinokio/api/comfyui.git/app/web
Workspace manager - Openning file hash dict
🦄🦄Loading: Workspace Manager (V2.1.0)
[comfyui_controlnet_aux] | INFO -> Using ckpts path: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/comfyui_controlnet_aux/ckpts
[comfyui_controlnet_aux] | INFO -> Using symlinks: False
[comfyui_controlnet_aux] | INFO -> Using ort providers: ['CUDAExecutionProvider', 'DirectMLExecutionProvider', 'OpenVINOExecutionProvider', 'ROCMExecutionProvider', 'CPUExecutionProvider', 'CoreMLExecutionProvider']
/home/sephia/pinokio/api/comfyui.git/app/custom_nodes/comfyui_controlnet_aux/node_wrappers/dwpose.py:26: UserWarning: DWPose: Onnxruntime not found or doesn't come with acceleration providers, switch to OpenCV with CPU device. DWPose might run very slowly
  warnings.warn("DWPose: Onnxruntime not found or doesn't come with acceleration providers, switch to OpenCV with CPU device. DWPose might run very slowly")

[rgthree] Loaded 40 fantastic nodes.
[rgthree] Will use rgthree's optimized recursive execution.

Warn!: xFormers is available (Attention)
Warn!: /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/utils/cpp_extension.py:1967: UserWarning: TORCH_CUDA_ARCH_LIST is not set, all archs for visible cards are included for compilation. 
If this is not desired, please set os.environ['TORCH_CUDA_ARCH_LIST'].
  warnings.warn(

Warn!: Traceback (most recent call last):
  File "/home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/utils/cpp_extension.py", line 2107, in _run_ninja_build
    subprocess.run(
  File "/home/sephia/pinokio/bin/miniconda/lib/python3.10/subprocess.py", line 526, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['ninja', '-v']' returned non-zero exit status 1.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/sephia/pinokio/api/comfyui.git/app/nodes.py", line 1931, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-3D-Pack/__init__.py", line 47, in <module>
    module = importlib.import_module(f".{nodes_filename}", package=__name__)
  File "/home/sephia/pinokio/bin/miniconda/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-3D-Pack/nodes.py", line 78, in <module>
    from Unique3D.scripts.project_mesh import multiview_color_projection, get_cameras_list
  File "/home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-3D-Pack/Gen_3D_Modules/Unique3D/scripts/project_mesh.py", line 98, in <module>
    pix2faces_renderer = Pix2FacesRenderer()
  File "/home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-3D-Pack/Gen_3D_Modules/Unique3D/scripts/project_mesh.py", line 70, in __init__
    self._glctx = dr.RasterizeGLContext(output_db=False, device=device)
  File "/home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/torch/ops.py", line 221, in __init__
    self.cpp_wrapper = _get_plugin(gl=True).RasterizeGLStateWrapper(output_db, mode == 'automatic', cuda_device_idx)
  File "/home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/torch/ops.py", line 118, in _get_plugin
    torch.utils.cpp_extension.load(name=plugin_name, sources=source_paths, extra_cflags=opts, extra_cuda_cflags=opts+['-lineinfo'], extra_ldflags=ldflags, with_cuda=True, verbose=False)
  File "/home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/utils/cpp_extension.py", line 1309, in load
    return _jit_compile(
  File "/home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/utils/cpp_extension.py", line 1719, in _jit_compile
    _write_ninja_file_and_build_library(
  File "/home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/utils/cpp_extension.py", line 1832, in _write_ninja_file_and_build_library
    _run_ninja_build(
  File "/home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/utils/cpp_extension.py", line 2123, in _run_ninja_build
    raise RuntimeError(message) from e
RuntimeError: Error building extension 'nvdiffrast_plugin_gl': [1/4] c++ -MMD -MF glutil.o.d -DTORCH_EXTENSION_NAME=nvdiffrast_plugin_gl -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1011\" -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/TH -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/THC -isystem /home/sephia/pinokio/bin/miniconda/include -isystem /home/sephia/pinokio/bin/miniconda/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++17 -DNVDR_TORCH -c /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/glutil.cpp -o glutil.o 
FAILED: glutil.o 
c++ -MMD -MF glutil.o.d -DTORCH_EXTENSION_NAME=nvdiffrast_plugin_gl -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1011\" -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/TH -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/THC -isystem /home/sephia/pinokio/bin/miniconda/include -isystem /home/sephia/pinokio/bin/miniconda/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++17 -DNVDR_TORCH -c /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/glutil.cpp -o glutil.o 
In file included from /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/glutil.cpp:14:
/home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/glutil.h:36:10: fatal error: EGL/egl.h: No such file or directory
   36 | #include <EGL/egl.h>
      |          ^~~~~~~~~~~
compilation terminated.
[2/4] c++ -MMD -MF rasterize_gl.o.d -DTORCH_EXTENSION_NAME=nvdiffrast_plugin_gl -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1011\" -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/TH -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/THC -isystem /home/sephia/pinokio/bin/miniconda/include -isystem /home/sephia/pinokio/bin/miniconda/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++17 -DNVDR_TORCH -c /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/rasterize_gl.cpp -o rasterize_gl.o 
FAILED: rasterize_gl.o 
c++ -MMD -MF rasterize_gl.o.d -DTORCH_EXTENSION_NAME=nvdiffrast_plugin_gl -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1011\" -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/TH -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/THC -isystem /home/sephia/pinokio/bin/miniconda/include -isystem /home/sephia/pinokio/bin/miniconda/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++17 -DNVDR_TORCH -c /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/rasterize_gl.cpp -o rasterize_gl.o 
In file included from /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/rasterize_gl.h:16,
                 from /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/rasterize_gl.cpp:9:
/home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/glutil.h:36:10: fatal error: EGL/egl.h: No such file or directory
   36 | #include <EGL/egl.h>
      |          ^~~~~~~~~~~
compilation terminated.
[3/4] c++ -MMD -MF torch_rasterize_gl.o.d -DTORCH_EXTENSION_NAME=nvdiffrast_plugin_gl -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1011\" -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/TH -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/THC -isystem /home/sephia/pinokio/bin/miniconda/include -isystem /home/sephia/pinokio/bin/miniconda/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++17 -DNVDR_TORCH -c /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/torch/torch_rasterize_gl.cpp -o torch_rasterize_gl.o 
FAILED: torch_rasterize_gl.o 
c++ -MMD -MF torch_rasterize_gl.o.d -DTORCH_EXTENSION_NAME=nvdiffrast_plugin_gl -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1011\" -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/TH -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/THC -isystem /home/sephia/pinokio/bin/miniconda/include -isystem /home/sephia/pinokio/bin/miniconda/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++17 -DNVDR_TORCH -c /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/torch/torch_rasterize_gl.cpp -o torch_rasterize_gl.o 
In file included from /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/torch/../common/rasterize_gl.h:16,
                 from /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/torch/torch_rasterize_gl.cpp:12:
/home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/torch/../common/glutil.h:36:10: fatal error: EGL/egl.h: No such file or directory
   36 | #include <EGL/egl.h>
      |          ^~~~~~~~~~~
compilation terminated.
ninja: build stopped: subcommand failed.


Warn!: Cannot import /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-3D-Pack module for custom nodes: Error building extension 'nvdiffrast_plugin_gl': [1/4] c++ -MMD -MF glutil.o.d -DTORCH_EXTENSION_NAME=nvdiffrast_plugin_gl -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1011\" -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/TH -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/THC -isystem /home/sephia/pinokio/bin/miniconda/include -isystem /home/sephia/pinokio/bin/miniconda/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++17 -DNVDR_TORCH -c /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/glutil.cpp -o glutil.o 
FAILED: glutil.o 
c++ -MMD -MF glutil.o.d -DTORCH_EXTENSION_NAME=nvdiffrast_plugin_gl -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1011\" -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/TH -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/THC -isystem /home/sephia/pinokio/bin/miniconda/include -isystem /home/sephia/pinokio/bin/miniconda/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++17 -DNVDR_TORCH -c /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/glutil.cpp -o glutil.o 
In file included from /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/glutil.cpp:14:
/home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/glutil.h:36:10: fatal error: EGL/egl.h: No such file or directory
   36 | #include <EGL/egl.h>
      |          ^~~~~~~~~~~
compilation terminated.
[2/4] c++ -MMD -MF rasterize_gl.o.d -DTORCH_EXTENSION_NAME=nvdiffrast_plugin_gl -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1011\" -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/TH -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/THC -isystem /home/sephia/pinokio/bin/miniconda/include -isystem /home/sephia/pinokio/bin/miniconda/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++17 -DNVDR_TORCH -c /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/rasterize_gl.cpp -o rasterize_gl.o 
FAILED: rasterize_gl.o 
c++ -MMD -MF rasterize_gl.o.d -DTORCH_EXTENSION_NAME=nvdiffrast_plugin_gl -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1011\" -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/TH -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/THC -isystem /home/sephia/pinokio/bin/miniconda/include -isystem /home/sephia/pinokio/bin/miniconda/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++17 -DNVDR_TORCH -c /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/rasterize_gl.cpp -o rasterize_gl.o 
In file included from /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/rasterize_gl.h:16,
                 from /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/rasterize_gl.cpp:9:
/home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/common/glutil.h:36:10: fatal error: EGL/egl.h: No such file or directory
   36 | #include <EGL/egl.h>
      |          ^~~~~~~~~~~
compilation terminated.
[3/4] c++ -MMD -MF torch_rasterize_gl.o.d -DTORCH_EXTENSION_NAME=nvdiffrast_plugin_gl -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1011\" -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/TH -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/THC -isystem /home/sephia/pinokio/bin/miniconda/include -isystem /home/sephia/pinokio/bin/miniconda/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++17 -DNVDR_TORCH -c /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/torch/torch_rasterize_gl.cpp -o torch_rasterize_gl.o 
FAILED: torch_rasterize_gl.o 
c++ -MMD -MF torch_rasterize_gl.o.d -DTORCH_EXTENSION_NAME=nvdiffrast_plugin_gl -DTORCH_API_INCLUDE_EXTENSION_H -DPYBIND11_COMPILER_TYPE=\"_gcc\" -DPYBIND11_STDLIB=\"_libstdcpp\" -DPYBIND11_BUILD_ABI=\"_cxxabi1011\" -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/torch/csrc/api/include -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/TH -isystem /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/torch/include/THC -isystem /home/sephia/pinokio/bin/miniconda/include -isystem /home/sephia/pinokio/bin/miniconda/include/python3.10 -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++17 -DNVDR_TORCH -c /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/torch/torch_rasterize_gl.cpp -o torch_rasterize_gl.o 
In file included from /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/torch/../common/rasterize_gl.h:16,
                 from /home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/torch/torch_rasterize_gl.cpp:12:
/home/sephia/pinokio/api/comfyui.git/app/env/lib/python3.10/site-packages/nvdiffrast/torch/../common/glutil.h:36:10: fatal error: EGL/egl.h: No such file or directory
   36 | #include <EGL/egl.h>
      |          ^~~~~~~~~~~
compilation terminated.
ninja: build stopped: subcommand failed.

WAS Node Suite: OpenCV Python FFMPEG support is enabled
WAS Node Suite Warning: `ffmpeg_bin_path` is not set in `/home/sephia/pinokio/api/comfyui.git/app/custom_nodes/was-node-suite-comfyui/was_suite_config.json` config file. Will attempt to use system ffmpeg binaries if available.
WAS Node Suite: Finished. Loaded 217 nodes successfully.

        "Art is the most powerful tool we have to connect with the world and express our individuality." - Unknown

Total VRAM 16071 MB, total RAM 31863 MB
pytorch version: 2.3.0+cu121
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4080 : cudaMallocAsync
[Allor]: 0 nodes were overridden.
[Allor]: 12 modules were enabled.
[Allor]: 98 nodes were loaded.
[Crystools INFO] Crystools version: 1.15.0
[Crystools INFO] CPU: 13th Gen Intel(R) Core(TM) i7-13700K - Arch: x86_64 - OS: Linux 5.15.0-116-generic
[Crystools INFO] Pynvml (Nvidia) initialized.
[Crystools INFO] GPU/s:
[Crystools INFO] 0) NVIDIA GeForce RTX 4080
[Crystools INFO] NVIDIA Driver: 535.183.01
### Loading: ComfyUI-Impact-Pack (V5.18.13)
### Loading: ComfyUI-Impact-Pack (Subpack: V0.6)
[Impact Pack] Wildcards loading done.
### Loading: ComfyUI-Manager (V2.48.1)
### ComfyUI Revision: 2393 [0a4c49c5] | Released on '2024-07-23'
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/alter-list.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/model-list.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/github-stats.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
Efficiency Nodes: Attempting to add Control Net options to the 'HiRes-Fix Script' Node (comfyui_controlnet_aux add-on)...Success!
Efficiency Nodes Warning: Failed to import python package 'simpleeval'; related nodes disabled.

[comfy_mtb] | INFO -> loaded 75 nodes successfuly
[comfy_mtb] | INFO -> Some nodes (5) could not be loaded. This can be ignored, but go to http://127.0.0.1:8188/mtb if you want more information.

Import times for custom nodes:
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/websocket_image_save.py
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/gcLatentTunnel.py
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-FBCNN
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/comfyui-yanc
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/masquerade-nodes-comfyui
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI_IPAdapter_plus
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-Video-Matting
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/Comfyui_joytag
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-Kolors-MZ
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI_UltimateSDUpscale
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/comfyui-browser
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/Derfuu_ComfyUI_ModdedNodes
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/comfyui-workspace-manager
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-Allor
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/Comfy_KepListStuff
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-BrushNet
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/rgthree-comfy
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/efficiency-nodes-comfyui
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/comfyui_tag_fillter
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-KJNodes
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/comfy_mtb
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-VideoHelperSuite
   0.0 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-Manager
   0.1 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-Impact-Pack
   0.2 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/comfyui_controlnet_aux
   0.4 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/was-node-suite-comfyui
   0.7 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-Inspyrenet-Rembg
   1.8 seconds (IMPORT FAILED): /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-3D-Pack
   2.2 seconds: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-Crystools

Starting server

To see the GUI go to: http://127.0.0.1:8188
FETCH DATA from: /home/sephia/pinokio/api/comfyui.git/app/custom_nodes/ComfyUI-Manager/extension-node-map.json [DONE]
```

**Comment by MrForExample** [comment]
Hi @manassm 
Seems you are missing some CPP header files in your system, you can check [Dockerfile](https://github.com/MrForExample/ComfyUI-3D-Pack/blob/main/Dockerfile) to see which library you are missing, but first and formost, make sure you have gcc g++ installed
```
sudo apt update
sudo apt install gcc g++
```

**Comment by manassm** [comment]
Thank you for the help. After a long trial and error, it seems like I made some progress, although I came across with bunch of new issues.
I will explain what I did to resolve it (partially) here for future reference to ask how to resolve the subsequent ones.
0. I am on Linux Mint 22 Cinnamon, and I installed ComfyUI through Pinokio frontend.
1. First, I installed miniconda (using this link: https://docs.anaconda.com/miniconda/miniconda-install/)
2. I went to the miniconda3 folder and ran this command: source bin/activate
3. Then I ran this command: eval "$(conda shell.bash hook)" && conda deactivate && conda deactivate && conda deactivate && conda activate base && source /home/<username>/pinokio/api/comfyui.git/app/env/bin/activate /home/<username>/pinokio/api/comfyui.git/app/env
4. (Not sure if it's relevant but) I updated pip using: pip install --upgrade pip
5. Then I ran: sudo apt update
sudo apt install gcc g++
6. I did: sudo apt update
sudo apt install mesa-common-dev libegl1-mesa-dev
7. I deleted and re-installed the 3D-pack extension through comfy manager.

Now I don't have 3D-pack nodes marked red anymore, which I guess means there were some progress. 
However, I have new set of problems regarding SF3D and UI, which I will mention on the separate discussion post. 

Thank you for help.
